backend/requirements.txt

```diff
@@ -0,0 +1,124 @@
+absl-py==0.8.1
```

Comment:
I think it would be better if
```
The requirements.txt was created by running python3 -m pip install tfx==0.21.0 && python3 -m pip freeze in the current python:3.5 image
```
is clearly documented inside code instead of in the PR.

Did you ever try pip-tools? Would a separate `requirements.in` file for direct deps and a compiled `requirements.txt` suit this model better?

Reply:
I just ran into the problem with unpinned dependencies, so fixed it in the most obvious way. I did't add the comment because it didn't seem to matter to me outside the context of the commit, because well, given subsequent runs of that command will yield a different requirements.txt.

Re/ pip-tools, not familiar with it. I don't do much python. Feel free to replace this PR with something better.

Reply:
I thought that comment is useful because we will probably install more dependencies/uninstall some dependencies from time to time. It will be a disaster to not know which ones are the actual direct dependencies.

Reply:
/assign @Ark-kun 
/assign @numerology 
I'm not familiar with python env too. They will be able to help.